docs/docs/glossary/node.md

```diff
@@ -3,32 +3,32 @@ title: Node.js
+যাই হোক, সেই জাভাস্ক্রিপ্ট এপ্লিকেশনগুলোর একটি বড় সীমাবদ্ধতা ছিল: সেগুলো ততটুকুই সম্পাদন করতে পারত যতটুকু রানটাইম তাদের অনুমোদন করত। ২০০৯ সালের আগে, রানটাইম প্রায় সবসময়ই ছিল একটি ওয়েব ব্রাউজার। তাই গুগল আংশিকভাবে একটি দ্রুতগতির ব্রাউজার তৈরির জন্য ক্রোমিয়াম প্রোজেক্ট গঠন করল। যার ফলাফল ছিল গুগল ক্রোম যেটা ২০০৮ সালে মুক্তি পায় এবং তাদের নতুন জাভাস্ক্রিপ্ট ইঞ্জিন: [V8](https://v8.dev/).
 
-A year later, Node.js made its debut as a standalone JavaScript runtime using the V8 engine.
+পরের বছর একটি স্বতন্ত্র জাভাস্ক্রিপ্ট রানটাইম হিসেবে Node.js এর অভিষেক ঘটে যা কিনা V8 ইঞ্জিন ব্যবহার করত। 
 
-Once you've installed Node.js, you can use it to run JavaScript from the [command line](/docs/glossary#command-line). Type `node` at a prompt to launch the Node.js interactive shell. Include the path to a JavaScript file to execute that script: e.g. `node /Users/gatsbyfan/hello-world.js`.
+আপনি একবার Node.js ইন্সটল করে ফেললে [command line](/docs/glossary#command-line) থেকে জাভাস্ক্রিপ্ট সম্পাদন করতে পারবেন। আপনি একটি ইন্টারেকটিভ শেলে 'node' টাইপ করুন। যে জাভাস্ক্রিপ্ট ফাইলটি আপনি সম্পাদন করতে চান তার পাথ 'node' কমান্ডের সাথে যোগ করে দিন। উদাহরণ - 'node /Users/gatsbyfan/hello-world.js'
 
-You will need to [install Node.js](/tutorial/part-zero/#install-nodejs-for-your-appropriate-operating-system) before using Gatsby. Gatsby is built using JavaScript, and requires the Node.js runtime.
+Gatsby ব্যবহারের পূর্বে আপনাকে Node.js ইন্সটল করে নিতে হবে [install Node.js](/tutorial/part-zero/#-install-nodejs-and-npm) Gatsby জাভাস্ক্রিপ্ট দিয়ে তৈরি এবং এর জন্য Node.js রানটাইম প্রয়োজন হয়। 
 
-Installing Node.js also installs [npm](/docs/glossary#npm), the Node.js _package manager_. A package manager is specialized software that lets you install and update modules and packages used in your project.
+Node.js ইন্সটল করলে তার সাথে [npm](/docs/glossary#npm), (Node.js এর প্যাকেজ ম্যানেজার) ও ইন্সটল হয়। প্যাকেজ ম্যানেজার এক ধরনের বিশেষ সফটওয়ার যা আপনার প্রোজেক্টে ব্যবহৃত বিভিন্ন মডিউল এবং প্যাকেজ আপডেট এবং  ইন্সটল করে দেয়।  
 
-You'll use npm to install Gatsby and its dependencies. Type `npm install -g gatsby-cli` at a command line prompt to install the Gatsby command line interface or CLI. The `-g` flag installs Gatsby globally, which means that you can use it by typing `gatsby` at a prompt. For example, you can use `gatsby new` to create a new Gatsby site.
+আপনি Gatsby এবং এর উপর নির্ভর করে এমন সবকিছু ইন্সটল করার জন্য npm ব্যবহার করবেন। Gatsby কমান্ড লাইন ইন্টারফেইস অথবা CLI ইন্সটল করার জন্য কমান্ড লাইন হিসেবে টাইপ করুন 'npm install -g gatsby-cli' । '-g' ফ্ল্যাগটি গ্লোবালি Gatsby ইন্সটল করে। এর মানে আপনি কমান্ড লাইনে gatsby লিখেই এটি ব্যবহার করতে পারবেন। উদাহরণস্বরূপ - 'gatsby new' টাইপ করে আপনি নতুন একটি Gatsby সাইট তৈরি করতে পারবেন। 
```

Comment:
নির্ভর করে -> নির্ভরশীল।
Please use grave accent(`) instead of  single quote(')